src/solver/type_getters.jl

```diff
@@ -879,6 +879,12 @@ function get_simulation(config::AtmosConfig)
     end
     @info "Prepared diagnostic callbacks: $s"
 
+    # One "cycle" is defined as the collection of iterations needed to go through all the
+    # (diagnostic) callbacks
```

Comment:
cycle, defined in `solve!` is a slightly different definition:

```
Run `step!` the least common multiple times
for all callbacks. i.e., all callbacks will
have been called at least `n_cycles` times.
```
In other words, if callback A and B are called every 3 and 4 steps, then a cycle is 11 steps. (maybe it'd be helpful to include that example in the docs). Would it be better to use a cycle in the way it's currently defined?

Reply:
Yes, my comment is wrong, but that's what I meant.